src/modules/evaluateExpression/evaluateExpression.ts

```diff
@@ -120,7 +96,7 @@ async function processPgSQL(queryArray: any[], queryType: string, connection: IC
     rowMode: queryType ? 'array' : '',
   }
   try {
-    const res = await connection.query(query)
+    const res = await connection.query(query.text, query.values)
```

Comment:
Change this back to just `connection.query(query)` -- it needs the whole object including `rowMode`

Reply:
Done. Great find! Wow, and sorry for missing that one there. Big mistake.